config/initializers/menus.rb

```diff
@@ -213,13 +213,13 @@
             caption: :label_news_plural,
             html: { class: 'icon2 icon-news' }
 
-  menu.push :new_news,
-            { controller: '/news', action: 'new' },
-            param: :project_id,
-            caption: :label_news_new,
-            parent: :news,
-            if: Proc.new { |p| User.current.allowed_to?(:manage_news, p.project) },
-            html: { class: 'icon2 icon-add' }
+  # menu.push :new_news,
+  #           { controller: '/news', action: 'new' },
+  #           param: :project_id,
+  #           caption: :label_news_new,
+  #           parent: :news,
+  #           if: Proc.new { |p| User.current.allowed_to?(:manage_news, p.project) },
+  #           html: { class: 'icon2 icon-add' }
```

Comment:
Did you intend to delete those lines and then accidentially hit the "comment out" key instead of delete?